src/server/pps/server/worker_rc_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 func mergeDefaultOptions(project, name string, add *workerOptions) *workerOptions {
 	base := &workerOptions{
-		rcName: strings.Join([]string{"pipeline", project, name, "v0"}, "-"),
+		rcName: strings.Join([]string{project, name, "v0"}, "-"),
```

Comment:
I'm pretty sure this doesn't matter (could be called "foobar" for all the tests care), but let's call pps.PipelineRCName here as well.